options.go

```diff
@@ -130,11 +130,12 @@ type Options struct {
 	// Set Verbose to true to enable more text logging.
 	Verbose bool `yaml:"verbose"`
 
-	// DEPRECATED: set `UseThrift` to true if you do not want gRPC
-	UseGRPC bool `yaml:"usegrpc"`
-
-	// Switch to
+	// Force the use of a specific transport protocol.
+	// If multiple are set to true, the top one is chosen.
+	// If none are set to true, GRPC is defaulted to.
 	UseThrift bool `yaml:"use_thrift"`
+	UseHttp bool `yaml:"use_http"`
```

Comment:
Do we plan to support HTTP/JSON in the go tracer in the future? I imagine there will be an additional option for use_http_json vs use_http_proto?

Reply:
I doubt it. Go has good support for proto and I don't this it's a big deal to rely on proto as a dependency. http/json is intended for languages that don't have good proto support (like for example javascript has in the past had bad proto support because it's very slow at bit packing operations).